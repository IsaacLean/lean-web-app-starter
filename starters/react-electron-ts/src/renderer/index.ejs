<!doctype html>
<html>
    <head>
        <meta charset="utf-8" />
        <title><%= htmlWebpackPlugin.options.title %></title>
        <meta name="viewport" content="width=device-width,initial-scale=1" />
    </head>
    <body>
        <div id="root"></div>
        We are using Node.js <span id="node-version"></span>, Chromium
        <span id="chrome-version"></span>, and Electron
        <span id="electron-version"></span>.
    </body>
</html>
