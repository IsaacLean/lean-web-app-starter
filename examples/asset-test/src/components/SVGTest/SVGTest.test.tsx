import renderer from 'react-test-renderer'

import SVGTest from '.'

test('SVGTest component matches snapshot', () => {
    const tree = renderer.create(<SVGTest />).toJSON()
    expect(tree).toMatchSnapshot()
})
