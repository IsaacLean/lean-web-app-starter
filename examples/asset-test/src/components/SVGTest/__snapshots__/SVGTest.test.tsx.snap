// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVGTest component matches snapshot 1`] = `
<div>
  <h2>
    SVG Test
  </h2>
  <img
    alt="React Logo"
    className="react-logo"
    src="test-file-stub"
  />
</div>
`;
